runtime/signals.c

```diff
@@ -291,6 +291,15 @@ void caml_request_minor_gc (void)
   caml_set_something_to_do();
 }
 
+value caml_check_gc_without_async_callbacks(value extra_root)
+{
+  CAMLparam1 (extra_root);
+  if (caml_requested_major_slice || caml_requested_minor_gc){
+    CAML_INSTR_INT ("force_minor/check_aosidjf@", 1);
```

Comment:
Eh?

Reply:
I wrote this line before I named the function and forgot to change it!